src/test/java/api/support/builders/ChangeDueDateRequestBuilder.java

```diff
@@ -0,0 +1,53 @@
+package api.support.builders;
+
+import static org.folio.circulation.resources.ChangeDueDateResource.DUE_DATE;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.JsonPropertyWriter;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import io.vertx.core.json.JsonObject;
+
+public class ChangeDueDateRequestBuilder implements Builder {
+  private final DateTime dueDate;
+  private final String loanId;
+
+  public ChangeDueDateRequestBuilder() {
+    this(null, DateTime.now(DateTimeZone.UTC));
+  }
+
+  private ChangeDueDateRequestBuilder(String loanId, DateTime dueDate) {
+
+    this.dueDate = dueDate;
+    this.loanId = loanId;
+  }
+
+  public ChangeDueDateRequestBuilder withDueDate(DateTime dateTime) {
+    return new ChangeDueDateRequestBuilder(loanId, dateTime);
+  }
+
+  public ChangeDueDateRequestBuilder forLoan(String loanId) {
+    return new ChangeDueDateRequestBuilder(loanId, dueDate);
+  }
+
+  public ChangeDueDateRequestBuilder forLoan(UUID loanId) {
+    return new ChangeDueDateRequestBuilder(loanId.toString(), dueDate);
+  }
+
+  public String getLoanId() {
+    return loanId;
+  }
+
+  public DateTime getDueDate() {
+    return dueDate;
+  }
+
+  @Override
+  public JsonObject create() {
+    final JsonObject request = new JsonObject();
+    JsonPropertyWriter.write(request, DUE_DATE, dueDate);
```

Comment:
I think statically importing the `write` method might reduce the noise in this statement

```suggestion
    JsonPropertyWriter.write(request, DUE_DATE, dueDate);
```